spec/paste.spec.js

```diff
@@ -139,6 +139,43 @@ describe('Pasting content', function () {
             }
         });
 
+        it('should add paragraphs to pasted plain-text', function () {
+            var editorEl = this.el,
+                editor = this.newMediumEditor('.editor', {
+                    delay: 200,
+                    paste: {
+                        forcePlainText: false,
+                        cleanPastedHTML: true
```

Comment:
So if you used the default options instead, you'd get the expected result without the code change:

default options:
```js
paste: {
    forcePlainText: true,
    cleanPastedHTML: false
}
```

Could you explain what the use case is that is causing this issue you're trying to fix?